e2e-tests/package.json

```
@@ -4,7 +4,7 @@
   "private": true,
   "description": "End-to-end tests for Apex Language Server Extension",
   "scripts": {
-    "test": "playwright test",
+    "test:e2e": "playwright test",
```

is this to exclude from other scripts?

---

Yes, it fails hard for me, stopping remaining tests.